proto/Makefile

```diff
@@ -12,23 +29,22 @@ get-deps:
 	go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
 	go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
 
-.PHONY: build-dir
-build-dir:
+build/proto.stamp: $(source_files)
+	rm -rf build/proto pkg
 	mkdir -p build/proto
+	# Protobuf generation.
+	for source in $(source_files) ; do protoc -I src "$$source" --go_out=plugins=grpc:build/proto ; done
```

Comment:
non-blocking: I rather liked that the old way would at least allow you to build the individual files in parallel; could that come back?

Reply:
Not without being able to pattern-match the target names.

The whole thing builds in like a second... I think this is probably fine.